### PR TITLE
chains: fix add ignore StreamingFunc

### PR DESCRIPTION
**Pull request by Abirdcfly**
Fellow #632 
Update #626

### CODE REVIEW

**Comment by Abirdcfly** [comment]
kindly ping @eliben, and the main branch look like lost some commit like #631

File: chains/options.go [hunk 1]
```diff
@@ -205,6 +207,9 @@ func getLLMCallOptions(options ...ChainCallOption) []llms.CallOption { //nolint:
 	if opts.repetitionPenaltySet {
 		chainCallOption = append(chainCallOption, llms.WithRepetitionPenalty(opts.RepetitionPenalty))
 	}
+	if opts.streamingFuncSet {
```
Comment (eliben) [inline chains/options.go:210]: I'm not sure we need the bool for the `func` options; the default for these is `nil`, which should be ok?

So you could just append the option regardless of the bool?

**Review by eliben — COMMENTED** [review]
Thanks, left a comment

**Comment by eliben** [comment]
Regarding lost commits, @tmc was fixing this issue last week so he will have the context on it here

File: chains/options.go (hunk 1 above)
Comment (Abirdcfly) [reply chains/options.go]: Updated.

**Review by eliben — APPROVED** [review]

**Comment by Abirdcfly** [comment]
> Regarding lost commits, @tmc was fixing this issue last week so he will have the context on it here

ping @tmc

**Pull request merged**